main.tf

```diff
@@ -29,12 +29,12 @@ module "object_storage" {
 module "service_accounts" {
   source = "./modules/service_accounts"
 
-  ca_certificate_secret  = var.ca_certificate_secret
-  license_secret         = var.license_secret
-  namespace              = var.namespace
-  ssl_certificate_secret = var.ssl_certificate_secret
-  ssl_private_key_secret = var.ssl_private_key_secret
-
+  ca_certificate_secret       = var.ca_certificate_secret
+  license_secret              = var.license_secret
+  namespace                   = var.namespace
+  ssl_certificate_secret      = var.ssl_certificate_secret
+  ssl_private_key_secret      = var.ssl_private_key_secret
+  override_service_account_id = var.override_service_account_id
```

Comment:
This is a little picky, but I wonder if we should standardize on `existing` as opposed to `override` for when a user is bringing their own resource/account. The AWS module doesn't use either, but Azure uses `existing` in one place (for a storage account).

Reply:
Totally not picky.  makes sense.  I'll get it changed over